apps/.storybook/config.js

```diff
@@ -69,26 +69,35 @@ function addStyleguideExamples(subcomponent) {
   }
 }
 
+const BLACKLIST = [
+  "code-studio/levels/contract_match.jsx",
+];
```

Comment:
I assume this just stops storybook from trying to load this file at all, since it's breaking things?